app/Entities/User.php

```diff
@@ -13,14 +13,20 @@ class User extends ShieldUser
      *
      * @return string
      */
-    public function renderAvatar(int $size=52)
+    public function renderAvatar(int $size=52, $random = false)
     {
+        // Generate random string for avatar placeholder
```

Comment:
Good catch - but I wouldn't force a random parameter here. Instead, if $idString is empty after checking for names, username, email, then give it a random string then. That way it's automatic. And a bit cleaner.